spec/facets/PartitionDatasetFacet.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema" : "https://json-schema.org/draft/2020-12/schema",
+  "$id" : "https://openlineage.io/spec/facets/1-0-0/PartitionDatasetFacet.json",
+  "$defs" : {
+    "PartitionDatasetFacet" : {
+      "allOf" : [ {
+        "$ref" : "https://openlineage.io/spec/1-0-2/OpenLineage.json#/$defs/DatasetFacet"
+      }, {
+        "type" : "object",
+        "properties" : {
+          "columns": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "properties": {
+                "columnName": {
```

Comment:
Given the containing property is already `columns`, I'd just go with `name` here.

Reply:
+1